examples/oneview_enclosure.yml

```diff
@@ -47,43 +50,45 @@
           hostname: '{{ enclosure_hostname }}'
           username: '{{ enclosure_username }}'
           password: '{{ enclosure_password }}'
-          name: 'Test-Enclosure'
+          name: '{{ enclosure_name  }}'
           licensingIntent : 'OneView'
           rackName: 'Rack-Name'
+      when: is_c7000 == true
```

Comment:
`is_c7000 == true` is redundant, just `is_c7000` is easier to read
This comment is aplicable for the other plays as well.